.github/workflows/release.yaml

```
@@ -55,16 +55,19 @@ jobs:
           export VIRTUAL_ENV=./venv
           python3.8 -m venv $VIRTUAL_ENV && source $VIRTUAL_ENV/bin/activate
           cd ./.github/workflows/release_scripts/  && pip install --quiet -r requirements.txt && python3.8 whitesource_vulnurability_checker.py
+      - name: Prepate Maven Settings
```

I assume this was supposed to say:
```suggestion
      - name: Prepare Maven Settings
```

---

Thanks Rohan, updated.